common/report/class.Report.php

```diff
@@ -107,6 +107,17 @@ public function __construct($type, $message = '', $data = null)
         $this->elements = [];
         $this->data = $data;
```

Comment:
Could you please move `__construct()` method to be the first method in this class

Reply:
Done